config/config.go

```diff
@@ -108,6 +108,10 @@ type Config struct {
 	// DischargeTokenTimeout is the maximum age a discharge token can
 	// get before it becomes invalid.
 	DischargeTokenTimeout DurationString `yaml:"discharge-token-timeout"`
+
+	// UseLocationForCookiePaths instructs if the Cookie Paths are to
+	// be set relative to the Location Path or not.
+	UseLocationForCookiePaths bool `yaml:"use-location-for-cookie-paths"`
```

Comment:
Because not every deployment uses the charm and this was a regression I think it would be better if the changed behaviour is on by default. Therefore I think you should reverse the meaning of the setting such that the path is added unless the setting is set.

Reply:
Makes sense :) Will do.